editor.planx.uk/src/@planx/components/List/Public/Fields.tsx

```diff
@@ -252,3 +256,76 @@ export const DateFieldInput: React.FC<Props<DateField>> = ({
     </InputLabel>
   );
 };
+
+export const MapFieldInput: React.FC<Props<MapField>> = (props) => {
+  const { formik, activeIndex, schema } = useListContext();
+  const {
+    id,
+    data: { title, fn, drawType, drawColor, drawMany },
+  } = props;
+
+  const teamSettings = useStore.getState().teamSettings;
+  const passport = useStore((state) => state.computePassport());
+
+  const [_features, setFeatures] = useState<Feature[] | undefined>(undefined);
+
+  useEffect(() => {
+    const geojsonChangeHandler = async ({ detail: geojson }: any) => {
+      if (geojson["EPSG:3857"]?.features) {
+        setFeatures(geojson["EPSG:3857"].features);
+        await formik.setFieldValue(
+          `userData[${activeIndex}]['${fn}']`,
+          geojson["EPSG:3857"].features,
+        );
+      } else {
+        // if the user clicks 'reset' on the map, geojson will be empty object, so set features to undefined
+        setFeatures(undefined);
+        await formik.setFieldValue(
+          `userData[${activeIndex}]['${fn}']`,
+          undefined,
+        );
+      }
+    };
+
+    const map: any = document.getElementById(id);
+
+    map?.addEventListener("geojsonChange", geojsonChangeHandler);
+
+    return function cleanup() {
+      map?.removeEventListener("geojsonChange", geojsonChangeHandler);
+    };
+  }, [setFeatures]);
+
+  return (
+    <InputLabel label={title} id={`map-label-${id}`} htmlFor={id}>
+      <ErrorWrapper
+        error={getIn(formik.errors, `userData[${activeIndex}]['${fn}']`)}
+        id={id}
+      >
+        <MapContainer environment="standalone">
+          {/* @ts-ignore */}
+          <my-map
+            id={id}
+            ariaLabelOlFixedOverlay={`An interactive map for plotting and describing ${schema.type.toLocaleLowerCase()}`}
+            height={400}
+            drawMode
+            drawMany={drawMany}
```

Comment:
note that `drawMany` won't actually do anything here yet until next map release !